integration-tests/simple-webapp/src/main/java/co/elastic/webapp/EchoServlet.java

```diff
@@ -0,0 +1,38 @@
+/*-
+ * #%L
+ * Elastic APM Java agent
+ * %%
+ * Copyright (C) 2018 - 2019 Elastic and contributors
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package co.elastic.webapp;
+
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class EchoServlet extends HttpServlet {
+
```

Comment:
It is a very good place to test all ServletInputStream APIs we support as it is tested on many servlet containers and Java versions.

Reply:
You are not requesting changes here, right?

Reply:
I suggest to test more ServletInputStream APIs, this is going to be tested on many implementations

Reply:
This is part of the integration test suite and thus tested for the `ServletInputStream` implementations of all supported application servers. What else should we test?

Reply:
The different `read` APIs